magento/magento2
# Fix DHL Quotes for Domestic Shipments when Content Type is set to Non-Document

### Description (*)
1. Updated label for Content Type configuration field to make it clear what it is.
2. Changed Allowable Methods (Non Doc and Doc) to always be displayed no
matter what the content type mode is. Makes no sense to display one or the other.
3. Changed the isDutiable function to only look at whether a shipment is
domestic or not. Previously it would return dutiable for everything
if the Content Type mode is non document (which causes problems for
domestic shipments in Non Document mode). Fixes #19485

### Fixed Issues (if relevant)
1. magento/magento2#19485:DHL Shipping Quotes fail for Domestic Shipments when Content Mode is "Non Documents"

### Manual testing scenarios (*)
1. Set Shipment Source to UK, SW1 1AA
2. Setup DHL module to obtain online quotes
3. Set Content Type to "Non Document"
4. Obtain quotes for destinations
4.1 UK, SW1 1AA (UK -> UK - Domestic - Documents - Non dutiable)
4.2 FR, 75001 (UK -> EU - Domestic - Documents - Non dutiable)
4.3 NO, 4032 (UK -> Europe - Intl - Non Documents - Dutiable)
4.4 US, 90210 (UK -> World - Intl - Non Documents - Dutiable)
5. Ensure quotes are received for all of the above regions.

### Contribution checklist (*)
